components/brave_wallet_ui/utils/string-utils.ts

```diff
@@ -6,7 +6,19 @@ export const toProperCase = (value: string) =>
     (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase())
 
 export const isRemoteImageURL = (url?: string) =>
-  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/')
+  url?.startsWith('http://') || url?.startsWith('https://') || url?.startsWith('data:image/') || url?.startsWith('ipfs://')
```

Comment:
Why `data/image/` is considered a remote image? Shouldn't the filter be named "isFrom(Distinct/Different)SiteImageURL"?

Reply:
@thypon I can rename the variable.